recipes/blockbuster/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+make
+mkdir $PREFIX/bin
+cp $SRC_DIR/blockbuster.x $PREFIX/bin
```

Comment:
Me too, me too. And sorry for the bin stuff. These recipes are generated automatically from Galaxy tool dependencies and I assumed that if tests pass everything should be fine.

Thanks again!